### PR TITLE
Sidebar Unordered Lists Restructure

### PR DESCRIPTION
**Pull request by yash37158**
**Notes for Reviewers**
Fixed sidebar unordered lists of restructure

This PR fixes #9485 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit da64f4c12b5187fcda1b675927740b365f2691cc at: https://6575876b836c4c044253395d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8961c33c6d1bad0b57cfccd9fca095355934f79f at: https://65758ead40c3bb0762950802--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@yash37158  are there any outstanding caveats worth noting in this PR? It seems to behave properly.

File: docs/_sass/sidebar.scss [hunk 1]
```diff
@@ -113,7 +113,7 @@
         font-size: 1.1rem;
         border-radius: 21px;
         background: var(--background-neumorphic);
-        padding: 8px 15px 8px 15px;
+        padding: 0px 15px 8px 15px;
```
Comment (vishalvivekm) [inline docs/_sass/sidebar.scss:116]:
```suggestion
        padding: 8px 15px 8px 15px;
```

Comment (vishalvivekm) [reply docs/_sass/sidebar.scss:116]: imo, let's keep the top padding as is, setting it to 0 results in:
Screenshot 2023-12-21 150114

**Comment by l5io** [comment]
🚀 Preview for commit ba4f9b0b318b1ee052f7255f69dec0b2f73d7be8 at: https://6584126e2473ebd53387eb60--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
One issue worth noting here is: 
as you click let's say Quick start, all the child list items are turning green. only the one which is currently being visited should be green instead. Here's clip:

https://github.com/meshery/meshery/assets/110674407/558c48a2-5cba-4156-bf72-7f008a53e724

**Comment by leecalcote** [comment]
Thank you, @vishalvivekm. @yash37158 ?

**Pull request merged**